src/base/trafficGraph.js

```diff
@@ -756,8 +761,10 @@ class TrafficGraph extends EventEmitter {
 
     _.each(this.nodes, (n) => { n.filtered = false; });
     _.each(this.connections, (c) => { c.filtered = false; });
-    this._updateConnectionFilters(filters);
-    this._updateNodeFilters(filters);
+    if (Object.keys(this.nodes).length > 1) {
```

Comment:
We need to also force counting on line 783 `if (node.connected)` since that will fail even though we are showing the node.  Probably makes sense to pull this check out into a boolean like: 
`const forceNodesVIsible = Object.keys(this.nodes).length > 1`